manifests/config.pp

```diff
@@ -0,0 +1,13 @@
+# SNMP config class to configure snmpd, snmptrapd, snmp utilities
+class snmp::config {
+
+  file { $::snmp::snmpd_config_file:
+    ensure  => present,
+    mode    => '0644',
+    owner   => 'root',
+    group   => 'root',
+    path    => $::snmp::snmpd_config_file,
+    content => template('snmp/RedHat/snmpd/snmpd.conf.erb'),
```

Comment:
I agree, I didn't use it at the moment, because there's a whole bunch more that should be added to the  config file as well for the template. This is just our basics.